### DISCUSSION TITLE
[Feature request] Please add top/bottom options for Google Cloud Monitoring queries (Builder)

### DISCUSSION THREAD

**Discussion by GedPL** [grafana/grafana#57755] [Data Sources]
HI,
Top/bottom functions limiting number of results are available in MQL used for querying Google Cloud Monitoring data source.
Would be nice to have them added to query builder for this data source.

Thanks !

Ged

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
